src/BSSelectIntegrationService/RetrieveMeshFile/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS installer-env
+
+COPY ./CaasIntegration/RetrieveMeshFile /src/dotnet-function-app
```

Comment:
not sure this will work as "./CaasIntegration/RetrieveMeshFile" isn't a valid file path. Is this file supposed to be working atm?